g1ant-language/introduction.md

```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+First thing you notice about G1ANT.Robot is that you communicate with it using scripts. These scripts are written in a special programming code called — surprise! — G1ANT.Language. You probably wonder what it is, why we chose to create it, and why we think this solution is superior to others. Here are the answers.
+
+## G1ANT.Language: What Is It?
```

Comment:
is it really true that we have our own programming language that is called "G1ANT.Language"?
for me it's weird as we don't have C#.Language, Python.Language or JQuery.Language languages. 

I think that G1ANT.Robot should be a software and G1ANT a programming language.

we're going to have a meeting about this but it's important to leave a comment here

Reply:
IMHO it's like MS would create language called "MS". If you would say "MS" no one would know if you mean the company or the language. G1ANT Language is fine for me.